modules/admin-portal/pages/management/user-management.adoc

```diff
@@ -61,14 +61,19 @@ Click image:delete_btn.png[] to delete a user.
 
 === Proxy groups
 
-Click the Proxy Group tab at the left to see all proxy groups:
+A proxy group is a way to manage users based on an aspect of their account identity set up as part of SSO or LDAP.
```

Comment:
"is a way" makes me feel like it's not telling me exactly what it is or what it does. Maybe just change to

> Proxy groups manage user access based on an aspect of their account identity set up as part of SSO or LDAP.